docs/01-dynamic-type-tests.md

```diff
@@ -44,7 +44,7 @@ function bar(x: ?string): string {
 }
 {% endhighlight %}
 
-Read more about <a href="docs/nullable-types.html">Maybe Types</a>.
+Read more about <a href="/docs/nullable-types.html">Maybe Types</a>.
```

Comment:
That site is pretty broken right now. There are a bunch of other absolute hrefs on this site already. Can we merge this as-is and I'll got back in and fix all the hrefs in a separate PR?